micromasters/settings.py

```diff
@@ -0,0 +1,151 @@
+"""
+Django settings for ui pp. This is just a harness type
+project for testing and interacting with the app.
+
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.8/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.8/ref/settings/
+
+"""
+import ast
+import os
+
+import dj_database_url
+import yaml
+
+CONFIG_PATHS = [
+    os.environ.get('MICROMASTERS_CONFIG', ''),
+    os.path.join(os.getcwd(), 'micromasters.yml'),
+    os.path.join(os.path.expanduser('~'), 'micromasters.yml'),
+    '/etc/micromasters.yml',
+]
+
+
+def load_fallback():
+    """Load optional yaml config"""
+    fallback_config = {}
+    config_file_path = None
+    for config_path in CONFIG_PATHS:
+        if os.path.isfile(config_path):
+            config_file_path = config_path
+            break
+    if config_file_path is not None:
+        with open(config_file_path) as config_file:
+            fallback_config = yaml.safe_load(config_file)
+    return fallback_config
+
+FALLBACK_CONFIG = load_fallback()
+
+
+def get_var(name, default):
+    """Return the settings in a precedence way with default"""
+    try:
+        value = os.environ.get(name, FALLBACK_CONFIG.get(name, default))
+        return ast.literal_eval(value)
+    except (SyntaxError, ValueError):
+        return value
+
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = get_var(
+    'SECRET_KEY',
+    '36boam8miiz0c22il@3&gputb=wrqr2plah=0#0a_bknw9(2^r'
+)
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = get_var('DEBUG', True)
+
+ALLOWED_HOSTS = get_var('ALLOWED_HOSTS', [])
+
+
+# Application definition
+
+INSTALLED_APPS = (
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    # Our INSTALLED_APPS
+    'ui'
```

Comment:
We would have that problem with any name, wouldn't it? I'm not saying that all ui has to go within this app, it's just a place to start writing code